cpp/webgl.cpp

```diff
@@ -15,6 +15,9 @@ namespace webgl {
 
 NAN_METHOD(init) {
 	
+	#ifdef __APPLE__
+	glewExperimental = GL_TRUE;
```

Comment:
Apfel über alles?

Can we release the kraken to all platforms? What could be the downsides?

Reply:
This is a hard requirement in MacOS, but I'm fine with setting it for everyone :-)

Reply:
what does it do on MacOS for you? i think it's of for all platforms then

> So what glewExperimental does is allow extension entry points to be loaded even if the extension isn’t present in the driver’s extensions string.

Reply:
https://stackoverflow.com/a/20822876